sdk/storage/azure-storage-file/azure/storage/file/_generated/operations/_directory_operations.py

```diff
@@ -213,6 +255,98 @@ def delete(self, timeout=None, cls=None, **kwargs):
             return cls(response, None, response_headers)
     delete.metadata = {'url': '/{shareName}/{directory}'}
 
+    def set_properties(self, file_attributes="none", file_creation_time="now", file_last_write_time="now", timeout=None, file_permission="inherit", file_permission_key=None, cls=None, **kwargs):
```

Comment:
Also for this one, would you like to verify that we don't want to use 'Preserve' as default value? :blush: